Documentation/Blazorise.Docs/Pages/News/2024-07-15-release-notes-160.razor

```diff
@@ -71,6 +71,17 @@
     Visit our docs for more information, <Anchor To="docs/extensions/chart-zoom">Chart Zoom Plugin</Anchor>
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    Video update
+</Heading>
+
+<Paragraph>
+    We have updatet our Video extension component to work with the new <Anchor To="https://www.vidstack.io/">VidStack</Anchor> library. Until now we have used a Plyr library, but it was slowly been replaced by a neweset and more modern VidStack library. So,  the obvious reason was for us also to make an update as it provides more features, and is much more stable. Not to mention that it also has many more features.
+</Paragraph>
+
+<Paragraph>
+    In this update we have mostly focused to make it work without any breaking changes, or introduction of new features. We have tried our best to polish it but as always there might be some edgecases that we have missed. In case you find any issue please report it to us and we will try to fix them as soon as possible.
```

Comment:
Despite XXXX bringing many new features, we've focused on migrating to this new library without any breaking changes. We have tried... bla bla...

Be on the lookout for new features on the next version (like... ) => tease about whatever features might be relevant

Reply:
PS:
We've added ProtectionServerCertificateUrl and thumbnails? Not sure if relevant to mention.